key-cert-provisioner/Makefile

```diff
@@ -55,11 +55,9 @@ $(BINDIR)/test-signer-$(ARCH):
 # BUILD IMAGE
 ###############################################################################
 .PHONY: image-all
-image-all: $(addprefix sub-image-,$(VALIDARCHES)) sub-image-fips-amd64
```

Comment:
why is FIPS taken out? pretty sure we still publish FIPS images for OSS

Reply:
Theoretically we do, but from what I've been able to tell we've literally never published a FIPS KSP image; there weren't any on quay.io or docker hub, so it seems as though it's never happened, and adding in FIPS building for KSP broke the test image that we also build but which does not build a FIPS version. @rene-dekker confirmed that we don't need it (but that if a user wanted it it's easy for them/us to build).